envoy/upstream/load_balancer.h

```diff
@@ -175,6 +229,8 @@ struct LoadBalancerParams {
   const PrioritySet& priority_set;
   // The worker local priority set of the local cluster.
   const PrioritySet* local_priority_set{};
+  // The thread local dispatcher.
+  Event::Dispatcher& dispatcher_;
```

Comment:
I could definitely split this out into its own PR which would remove maybe 10-15 test files.

Reply:
That also sounds nice because it would mean that the remaining PR has fewer moving pieces to keep mental track of, I think.